File: Elixir/abc197_a/main.ex
defmodule Main do
  def main do
    IO.gets("")
    |> String.trim()
    |> solve()
    |> IO.puts()
  end

  def solve(str) do
    String.graphemes(str)
    |> then(fn [first | rest] -> Enum.join(rest) <> first end)
  end
end

Main.main()

File: TypeScript/abc197_a/main.ts
async function getInputToString(): Promise<string> {
    const input = await new Response(Deno.stdin.readable).text();
    return input.trim();
}

function solve(input: string): string {
    const first = input.slice(0,1);
    const rest = input.slice(1);
    return rest + first;
}

const inputText = await getInputToString();

console.log(solve(inputText));

export{};

